sdk/search/search-documents/src/indexModels.ts

```diff
@@ -37,6 +37,26 @@ export interface SearchIndexingBufferedSenderOptions {
    * Indicates if autoFlush is enabled.
    */
   autoFlush?: boolean;
+  /**
+   * Initial Batch Document Count
```

Comment:
can I get some more context on what this means - is this the default window size? e.g. after getting this many documents, we try to submit a batch? or something else?

Reply:
Yes. It is the number of documents.  Ted wanted to specifically add 'initial' to convey the idea that this might change in the future. (What might change? we may add the ability to modify the count in the middle or we may change the logic to threshold count i.e. size of the documents instead of number of documents)

Reply:
Added more descriptive comment in the code.